cbmackintosh/rancid-tomatillos
# Feature/data error handling

## What does this PR do?
 - Adds a componentDidMount() method which calls network request for all movies in the database
 - Adds error handling for 500 status codes
 - Adds error handling for other status codes (@sertmer  @letakeane : I'm not sure how to test this since I can't shut down the backend server, do you have any suggestions?)
 - Displays data fetched via the API to the DOM with conditional rendering
 - Organizes files into appropriate folders
 
## How should it be tested?
 - Run the git command `git switch Feature/data-error-handling` and then run the app
 - Make sure all movies are loading
 - Make sure previous functionality is still there
 - Mess up the url in the fetch request to see the error message

SCOTT & LETA: I remember seeing one of you demonstrate setting `this.state.error` for failed fetch requests, but the message you displayed to the user was different than the message found in the React dev tools. I can't remember how you did it though. Care to refresh my memory? 😅🦑
